### ISSUE TITLE
Skipping stuff thats in Pickit

### ISSUE THREAD

**Issue by sollaf** [kolton/d2bot-with-kolbot#1001]
Hey. So I know how to do pickit etc...

Though I've seen the bot skip grey items. Lets take an example... 

It has skipped several grey armors and sacred rondaches. 

I paused the bot and picked those up but they werent what I needed so I dropped them again.

Does the bot actually know that the stats on the item without picking it up? It seems like that since when I pick them up they do not have the stats they should for the bot to pick it up?

**Comment by 5noop** [comment]
> Does the bot actually know that the stats on the item without picking it up

Yes, it will pick the whites in your pickit don't worry about it.

@semirotta: you definitely have a ko somewhere in your pickit, or a cube config that needs ko

**Comment by semirotta** [comment]
I was thinking about cube config at first, but then I noticed that I didnt have any kind of cubing turned on at that point. I searched through the kolton file and I have Ko now in the list but I made it there myself, because I was trying to make it not to collect them :'D

**Comment by JohnGoblikon** [comment]
How do you pause the bot mid run?

**Comment by semirotta** [comment]
You dont. Type in chat .stop and .start
If you type .stop, it'll take few seconds and the game shuts down and restarts.

**Comment by aim2kill** [comment]
to pause the bot you need to press ctrl+numlock

**Comment by semirotta** [comment]
Thank you, I was asking from a friend who introduced this bot to me about it and he did not know so he just told me about the .stop and .start. That will help a lot.

**Comment by aim2kill** [comment]
Np mate just helping 👍 btw make sure you unpause the bot in the same place(location)you unpause

**Comment by semirotta** [comment]
exactly same spot as I paused? like if im 2 steps left, my computer blows up?

**Comment by aim2kill** [comment]
If u pause at chaos/tristam,etc dont unpause at town, you need to go back to chaos/tristam,etc

**Comment by semirotta** [comment]
Ah ok thank you, thought I must be in same spot exactly :'D ahhah I feel silly now.

**Comment by JohnGoblikon** [comment]
> @semirotta you wrong you are able to pause you just dont know it
> to pause the bot you need to press ctrl+numlock and again to unpause
> 
> (edit) press ESC to close the console 1st if its open

You are my hero. ty kind sir or madame. Do you know of a list of commands. I've googled in game commands for kolbot but all I get is commands for follower.
